src/client/xmlClient.ts

```diff
@@ -14,6 +14,17 @@ import * as Telemetry from '../telemetry';
 import { ClientErrorHandler } from './clientErrorHandler';
 import { activateTagClosing, AutoCloseResult } from './tagClosing';
 
+export const XML_SUPPORTED_LANGUAGE_IDS = ['xml', 'xsl', 'dtd', 'svg'];
+
+const XML_DOCUMENT_SELECTOR: DocumentSelector =
+  XML_SUPPORTED_LANGUAGE_IDS.map(function (langId) {
+    return { scheme: 'file', language: langId }
+  }).concat(
+    XML_SUPPORTED_LANGUAGE_IDS.map(function (langId) {
+      return { scheme: 'untitled', language: langId }
+    }));
```

Comment:
I think you should be able to make this a bit more compact and easy to read using the arrow syntax eg.:

```typescript
.map(langId => { scheme: 'file', language: langId })
```

Reply:
I tried it but it doesn't work since it is a finction in the map.

Reply:
It seems like it works if you wrap the return object in `(` `)`